util/cron/test-linux64-python36.bash

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+#
+# Test default configuration with `make check`, on linux64, with python 3.6
```

Comment:
Looks like the python3 variants are running the hellos, not `make check`

Reply:
Good catch